packages/features/eventtypes/lib/getPublicEvent.ts

```diff
@@ -291,6 +292,7 @@ export const getPublicEvent = async (
     profile: getProfileFromEvent(eventWithUserProfiles),
     users,
     entity: {
+      fromRedirectOfNonOrgLink,
```

Comment:
Send it back so that react query can sent it again and cached response can be served by react query.